### PR TITLE
validation unit test mapArrayValuesToNull

### PR DESCRIPTION
**Pull request by OgunyemiO**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EXUI-1121

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.purger.ts [hunk 1]
```diff
@@ -309,10 +309,18 @@ export class FieldsPurger {
...
    */
   public mapArrayValuesToNull(array: any[]): any[] {
-    return array.map(element => {
-      return typeof element === 'object'
-        ? Object.assign({}, ...Object.keys(element).map(k => ({ [k]: null })))
-        : null;
-    });
+    if (array && array.length > 0) {
+      return array.map(element => {
+        if (element !== undefined || element === null) {
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.purger.ts:314]: Not sure what element === null is adding here. Would work the same without that - or am I missing something

**Review by andywilkinshmcts — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.purger.ts (hunk 1 above)
Comment (OgunyemiO) [reply projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.purger.ts:314]: @andywilkinshmcts sometimes the value returned is NULL and not just undefined. In such situation, the code fails and that's why I added that extra check.

**Review by RiteshHMCTS — COMMENTED** [review]
@OgunyemiO  can you add more unit tests to pass the threshold limit of 80% coverage

**Comment by OgunyemiO** [comment]
The issue raised in this ticket is now covered with the fix in https://tools.hmcts.net/jira/browse/EXUI-1112

**Pull request closed without merging**